### DISCUSSION TITLE
enabling cluster grouping

### DISCUSSION THREAD

**Discussion by jigneshmpatel** [redis/lettuce#2858] [Q&A]
Is there any way to enable cluster grouping so that as a subscriber if the application is in cluster mode, only one node processes the subscription message?

**Comment by tishun** [comment]
Depending on how your cluster is set up you could use [PUBSUB SHARDCHANNELS](https://redis.io/docs/latest/commands/pubsub-shardchannels/)
Sharded PubSub is still under development for the lettuce driver, see #2762.

Does that help?
